File: tests/src/main/java/com/github/klee0kai/test/inject/Forest.java
package com.github.klee0kai.test.inject;

import com.github.klee0kai.stone.Stone;
import com.github.klee0kai.test.inject.di.ForestComponent;

public class Forest {

    public static ForestComponent DI;


    public void create() {
        DI = Stone.createComponent(ForestComponent.class);
    }

}

File: test_kotlin/src/main/kotlin/com/github/klee0kai/test/inject/Forest.kt
package com.github.klee0kai.test.inject

import com.github.klee0kai.stone.Stone
import com.github.klee0kai.test.inject.di.ForestComponent

class Forest {
    fun create() {
        DI = Stone.createComponent(
            ForestComponent::class.java
        )
    }

    companion object {
        var DI: ForestComponent? = null
    }
}